### ISSUE TITLE
Security dependencies

### ISSUE THREAD

**Issue by rachaelshaw** [balderdashy/sails#3073]
From @npoirier-palo (originally posted in the sails-docs repo [here](https://github.com/balderdashy/sails-docs/issues/492)):

> Hello

> We noticed few problem with version 0.11 and  dependencies contains vulnerabilites detect by nsp (https://github.com/nodesecurity/nsp) details

> Name          Installed       Patched         Vulnerable Dependency
> qs                 0.5.6          >= 1.x             grunt-contrib-watch@0.5.3 > tiny-lr@0.0.4 > qs@0.5.6
> serve-index     1.1.6       >=1.6.3           sails@0.11.1 > express@3.16.0 > connect@2.25.0 > serve-> index@1.1.6
> serve-static    1.5.4        >=1.7.2           sails@0.11.1 > express@3.16.0 > connect@2.25.0 > serve-> static@1.5.4
> send              0.8.1         >= 0.8.4          sails@0.11.1 > express@3.16.0 > send@0.8.1
> semver          3.0.1         >=4.3.2           sails@0.11.1 > express-handlebars@1.0.3 > semver@3.0.1
> semver          2.2.1         >=4.3.2           sails@0.11.1 > semver@2.2.1

> For the next release is-it possible to fix the version ?

> Thx and congratulations for the real good job with sailsjs !

> Regards

**Comment by sailsbot** [comment]
Thanks for posting, @rachaelshaw.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
